### ISSUE TITLE
Getting 403 Forbidden at some png calls

### ISSUE THREAD

**Issue by iwollmann** [Microsoft/PowerBI-JavaScript#260]
I'm getting these errors with some reports, but the strange thing is that the report is being displayed correctly.

![image](https://user-images.githubusercontent.com/997343/31040190-97c3cfe6-a55a-11e7-8f68-c75ebdf81cfa.png)

What could this be?

**Comment by nicksav** [comment]
Might be also related?
![image](https://user-images.githubusercontent.com/9266365/31372800-d11737f0-adda-11e7-81f0-d56cc95a76b9.png)

**Comment by iwollmann** [comment]
I don't think so. I discovered that this only happens with static images on cloned reports, it seems the new token from copied report does not have access to images from the original report.

**Comment by ali-hamud** [comment]
Hi @iwollmann 
Do you have a consestent repro? This will help us solve the issue faster.

Thanks,
Ali Hamud

**Comment by iwollmann** [comment]
Yes, just follow the steps I mentioned earlier. But it seems that is not a problem with this repo but it with the powerbi.
